 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 318-090-404
127.0.0.1 - - [14/Oct/2020 10:10:00] "[37mGET / HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 10:10:00] "[36mGET /static/styles/base.css HTTP/1.1[0m" 304 -
127.0.0.1 - - [14/Oct/2020 10:10:00] "[37mGET /static/images/favicon.ico HTTP/1.1[0m" 200 -
 * Detected change in '/Users/davidhaase/Documents/Projects/logos/app/main/views.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 103-549-827
127.0.0.1 - - [14/Oct/2020 20:28:23] "[35m[1mGET / HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/davidhaase/Documents/Projects/logos/logos.py", line 5, in <module>
    app = create_app(os.getenv('FLASK_CONFIG') or 'default')
  File "/Users/davidhaase/Documents/Projects/logos/app/__init__.py", line 44, in create_app
    from .main import main as main_blueprint
  File "/Users/davidhaase/Documents/Projects/logos/app/main/__init__.py", line 28, in <module>
    from . import views, errors
File "/Users/davidhaase/Documents/Projects/logos/app/main/views.py", line 439
    @main.route('/themodels/<engine_name>', methods=['GET', 'POST'])
    ^
SyntaxError: invalid syntax
127.0.0.1 - - [14/Oct/2020 20:28:23] "[37mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:28:23] "[37mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:28:23] "[37mGET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:28:23] "[37mGET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:28:23] "[37mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:28:23] "[37mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:29:59] "[37mGET / HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:29:59] "[37mGET /static/images/favicon.ico HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:30:02] "[33mGET /translationhistory HTTP/1.1[0m" 404 -
127.0.0.1 - - [14/Oct/2020 20:30:03] "[33mGET /static/images/icons/favicon.ico HTTP/1.1[0m" 404 -
 * Detected change in '/Users/davidhaase/Documents/Projects/logos/app/main/views.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 103-549-827
127.0.0.1 - - [14/Oct/2020 20:30:42] "[37mGET / HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:30:42] "[37mGET /static/images/favicon.ico HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:30:45] "[35m[1mGET /translationhistory HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/davidhaase/Documents/Projects/logos/app/main/views.py", line 433, in translationhistory
    table_of_translation_history=table_of_translation_history)
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/templating.py", line 140, in render_template
    ctx.app,
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/Users/davidhaase/Documents/Projects/logos/app/templates/translationhistory.html", line 2, in top-level template code
    {% import "bootstrap/wtf.html" as wtf %}
  File "/Users/davidhaase/Documents/Projects/logos/app/templates/base.html", line 1, in top-level template code
    {% extends "bootstrap/base.html" %}
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/Users/davidhaase/Documents/Projects/logos/app/templates/base.html", line 69, in block "content"
    {% block page_content %}{% endblock %}
  File "/Users/davidhaase/Documents/Projects/logos/app/templates/translationhistory.html", line 8, in block "page_content"
    <p>Page last refreshed: {{ moment(current_time).format('LLLL') }}
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask_moment.py", line 153, in format
    return self._render("format", format=(fmt or ''), refresh=refresh)
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask_moment.py", line 139, in _render
    t = self._timestamp_as_iso_8601(self.timestamp)
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask_moment.py", line 135, in _timestamp_as_iso_8601
    return timestamp.strftime('%Y-%m-%dT%H:%M:%S' + tz)
jinja2.exceptions.UndefinedError: 'current_time' is undefined
127.0.0.1 - - [14/Oct/2020 20:30:45] "[37mGET /translationhistory?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:30:45] "[37mGET /translationhistory?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:30:45] "[37mGET /translationhistory?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:30:45] "[37mGET /translationhistory?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:30:45] "[37mGET /translationhistory?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:30:45] "[37mGET /translationhistory?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/Users/davidhaase/Documents/Projects/logos/app/main/views.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 103-549-827
127.0.0.1 - - [14/Oct/2020 20:31:09] "[35m[1mGET /translationhistory HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/Users/davidhaase/Documents/Projects/logos/venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/davidhaase/Documents/Projects/logos/logos.py", line 5, in <module>
    app = create_app(os.getenv('FLASK_CONFIG') or 'default')
  File "/Users/davidhaase/Documents/Projects/logos/app/__init__.py", line 44, in create_app
    from .main import main as main_blueprint
  File "/Users/davidhaase/Documents/Projects/logos/app/main/__init__.py", line 28, in <module>
    from . import views, errors
File "/Users/davidhaase/Documents/Projects/logos/app/main/views.py", line 434
    urrent_time=datetime.utcnow())
              ^
SyntaxError: invalid syntax
127.0.0.1 - - [14/Oct/2020 20:31:09] "[37mGET /translationhistory?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:31:09] "[37mGET /translationhistory?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:31:09] "[37mGET /translationhistory?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:31:09] "[37mGET /translationhistory?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:31:09] "[37mGET /translationhistory?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:31:09] "[37mGET /translationhistory?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:31:26] "[37mGET /translationhistory HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:31:27] "[37mGET /static/images/favicon.ico HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:32:06] "[37mGET /translationhistory HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:32:06] "[37mGET /static/images/favicon.ico HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:32:54] "[37mGET / HTTP/1.1[0m" 200 -
2020-10-14 20:33:30.251262: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-10-14 20:33:30.268413: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x7ffb7d2e1630 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-10-14 20:33:30.268446: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
127.0.0.1 - - [14/Oct/2020 20:33:31] "[32mPOST / HTTP/1.1[0m" 302 -
127.0.0.1 - - [14/Oct/2020 20:33:32] "[37mGET / HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:36:26] "[37mGET / HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:36:26] "[37mGET /static/images/favicon.ico HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:37:16] "[37mGET / HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:37:50] "[32mPOST / HTTP/1.1[0m" 302 -
127.0.0.1 - - [14/Oct/2020 20:37:50] "[37mGET / HTTP/1.1[0m" 200 -
127.0.0.1 - - [14/Oct/2020 20:37:56] "[37mGET /translationhistory HTTP/1.1[0m" 200 -
